bin/mpdevil

```diff
@@ -3391,16 +3391,20 @@ class AudioFormat(Gtk.Box):
 			self._format_label.set_markup("<small>"+ClientHelper.convert_audio_format(audio_format)+"</small>")
 
 	def _on_bitrate(self, emitter, brate):
-		self._brate_label.set_text(locale.str(brate))
+		# handle unknown bitrates: https://github.com/MusicPlayerDaemon/MPD/issues/428#issuecomment-442430365
+		if brate == "0":
+			self._brate_label.set_text("-- kb/s • ")
```

Comment:
Here is just a small typographical "issue" I'd use the same dashes as in the `SeekBar` namely `U+2013` "–". And I think one of those should be enough.